Realm/RLMRealm.mm

```diff
@@ -447,6 +447,12 @@ + (instancetype)realmWithConfiguration:(RLMRealmConfiguration *)configuration
         }
     }
 
+    if (configuration.seedFilePath && ![RLMRealm fileExistsForConfiguration:configuration]) {
+        [[NSFileManager defaultManager] copyItemAtURL:configuration.seedFilePath
```

Comment:
We need to be checking the return value of this.

Reply:
What should we do if it returns false?

Reply:
I have added that it will throw if the return value is `NO`